authmanager/authmanager.go

```diff
@@ -14,10 +14,11 @@ import (
 )
 
 const (
-	githubAPITokenMsg  = "Please go to https://github.com/settings/tokens, create a read-only access token & paste it here:\nToken: "
-	gitlabAPITokenMsg  = "Please go to %s/profile/personal_access_tokens, create a read-only access token & paste it here:\nToken: "
-	pivotalAPITokenMsg = "Please go to https://www.pivotaltracker.com/profile, create a new API token & paste it here:\nToken: "
-	redmineAPITokenMsg = "Please go to %s/my/account, create a new API token & paste it here:\nToken: "
+	githubAPITokenMsg   = "Please go to https://github.com/settings/tokens, create a read-only access token & paste it here:\nToken: "
+	gitlabAPITokenMsg   = "Please go to %s/profile/personal_access_tokens, create a read-only access token & paste it here:\nToken: "
+	pivotalAPITokenMsg  = "Please go to https://www.pivotaltracker.com/profile, create a new API token & paste it here:\nToken: "
+	redmineAPITokenMsg  = "Please go to %s/my/account, create a new API token & paste it here:\nToken: "
+	youtrackAPITokenMsg = "Please go to '%s/youtrack/users/me', create a new API token & paste it here (More info - 'https://www.jetbrains.com/help/youtrack/standalone/Manage-Permanent-Token.html'):\nToken: "
```

Comment:
All these constants are actually not used anymore as this is now done via the `IssueTracker.TokenAcquisitionInstructions()` function

Reply:
Could you please remove them?

Reply:
They are removed.